docs/user/monitor_experiments_with_tensorboard.md

```diff
@@ -0,0 +1,132 @@
+# Monitor Your Experiments with TensorBoard
+
+[TensorBoard](https://www.tensorflow.org/tensorboard) is a powerful tool that
+allows you to monitor and visualize an experiment.
+This section introduces how to set up TensorBoard logging in garage, and how to
+view/share your TensorBoard logs.
+
+## Prerequisite
+
+The TensorBoard package should be installed upon garage's installation.
+It can be also manually installed via:
+
+```bash
+pip install tensorboard
+```
+
+### Setup Your Experiment
+
+Use `@wrap_experiment` to setup your experiment.
+This will automatically setup TensorBoard output and allocate an experiment
+directory for you.
+
+Find more about how to use `@wrap_experiment` [here](https://github.com/rlworkgroup/garage/blob/master/docs/user/experiments.rst).
```

Comment:
Relative link could also be used here. For example, `[here](experiments)`.

Reply:
Got it. Thank you!

Reply:
It seems that `[here](experiments.rst)` doesn't work when html generated. `[here](experiments)` works for me. You can see docs/readthedocs.org:garage in the check to preview it.

Reply:
Changed to `[here](experiments)` and it works now!